buddyboss/buddyboss-platform
# Activity feed - distorted small images

### All Submissions:


Fixes #1084 

### How to test the changes in this Pull Request:

1. Go to activity feed
2. Add post with small image
3. Scroll down to see result

### Proof Screenshots or Video
http://i.imgur.com/F9Ekpzd.png

### Other information:
